packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -646,6 +703,61 @@ function hexToRgbPartial(hexValue: string): string {
     return `${red}, ${green}, ${blue}`;
 }
 
+function createDropShadowTokens(
+    dropShadowTokenName: string,
+    colorFunction: (element: HTMLElement) => string,
+    xOffset: string,
+    yOffset: string,
+    blurSize: string
+): readonly [
+        CSSDesignToken<string>,
+        CSSDesignToken<string>,
+        CSSDesignToken<string>,
+        CSSDesignToken<string>,
+        CSSDesignToken<string>
+    ] {
+    if (
+        dropShadowTokenName === ''
+        || xOffset === ''
+        || yOffset === ''
+        || blurSize === ''
+    ) {
+        throw new Error(
+            'createDropShadowTokens parameter unexpectedly set to empty string'
+        );
+    }
+
+    const dropShadowToken = DesignToken.create<string>(
+        styleNameFromTokenName(dropShadowTokenName)
+    ).withDefault(
+        (element: HTMLElement) => `${xOffset} ${yOffset} ${blurSize} ${colorFunction(element)}`
+    );
+
+    const dropShadowColorToken = DesignToken.create<string>(
+        styleNameFromTokenName(`${dropShadowTokenName}-color`)
+    ).withDefault((element: HTMLElement) => colorFunction(element));
+
+    const dropShadowXOffsetToken = DesignToken.create<string>(
+        styleNameFromTokenName(`${dropShadowTokenName}-x-offset`)
+    ).withDefault(`${xOffset}`);
+
+    const dropShadowYOffsetToken = DesignToken.create<string>(
+        styleNameFromTokenName(`${dropShadowTokenName}-y-offset`)
+    ).withDefault(`${yOffset}`);
+
+    const dropShadowBlurSizeToken = DesignToken.create<string>(
+        styleNameFromTokenName(`${dropShadowTokenName}-blur-size`)
+    ).withDefault(`${blurSize}`);
```

Comment:
for this and the few above can just return the string instead of using in a template string

Reply:
Because I removed all the subparts of the shadow tokens, I also removed the `createDropShadowTokens()` function. This code no longer exists.